terra_layer/models.py

```diff
@@ -216,6 +216,33 @@ def save(self, **kwargs):
     def __str__(self):
         return f"Layer({self.id}) - {self.name}"
 
+    @transaction.atomic()
+    def replace_source(self, source, fields_matches={}):
```

Comment:
```suggestion
    def replace_source(self, new_source, fields_matches={}):
```

Reply:
Since I amend the commit, I changed it too